### PR TITLE
Snowbridge Rococo deployment updates

### PR DESCRIPTION
**Pull request by claravanstaden**
- Includes the introduction of the `fast-runtime` feature, which cleans up our features
- Updates beacon client fork versions config to Sepolia's versions
- Cleanup of AgentIdOf

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4792795

**Review by bkontur — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4793097

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4793552

**Comment by bkontur** [comment]
Before merge and deploy, you need to change RuntimeVersion for BHR  1,005,001 to 1,005,002 in lib.rs

File: cumulus/polkadot-parachain/Cargo.toml [hunk 1]
```diff
@@ -161,3 +161,6 @@ try-runtime = [
 	"shell-runtime/try-runtime",
 	"sp-runtime/try-runtime",
 ]
+fast-runtime = [
+	"bridge-hub-rococo-runtime/fast-runtime",
+]
```
Comment (bkontur) [inline cumulus/polkadot-parachain/Cargo.toml:166]:
```suggestion
fast-runtime = []
```
I think it should work just like this (without defaulting `"bridge-hub-rococo-runtime/fast-runtime"`),
@alistair-singh @claravanstaden could you, please, try it if it works for you local testing?

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4861572

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4861860

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4863666

File: cumulus/polkadot-parachain/Cargo.toml (hunk 1 above)
Comment (claravanstaden) [reply cumulus/polkadot-parachain/Cargo.toml:172]: @bkontur it doesn't work, it seems. The `fast-runtime` feature should be propagated to the bridge hub rococo runtime for it to work.

**Review by vgeddes — APPROVED** [review]
+1

**Review by acatangiu — APPROVED** [review]
Looks good!

**Pull request merged**